docs/source/debug_accuracy.md

```diff
@@ -0,0 +1,110 @@
+# Step by step example how to debug accuracy with Neural Insights
+1. [Introduction](#introduction)
+2. [Environment preparation](#environment-preparation)
+3. [Running the quantization](#running-the-quantization)
+4. [Analyzing the result of quantization](#-analyzing-the-result-of-quantization)
+5. [Analyzing weight histograms](#-analyzing-weight-histograms)
+
+# Introduction
+In this instruction accuracy issue will be debugged using Neural Insights. TensorFlow Inception_v3 model will be used as an example. It will be quantized and the results will be analyzed to find the cause of the accuracy loss.
+
+# Environment preparation
+## Install Intel® Neural Compressor and Neural Insights from source
+First you need to install Intel® Neural Compressor.
+```shell
+# Install Neural Compressor
+git clone https://github.com/intel/neural-compressor.git
+cd neural-compressor 
+pip install -r requirements.txt 
+python setup.py install
+
+# Install Neural Insights
+pip install -r neural_insights/requirements.txt
+python setup.py install neural_insights
+```
+
+## Install requirements for inception_v3
+```shell
+cd examples/tensorflow/object_detection/tensorflow_models/quantization/ptq
```

Comment:
The model path should be `examples/tensorflow/image_recognition/tensorflow_models/inception_v3/quantization/ptq`, there is something wrong with example readme, just ask owner to fix.